src/radical/pilot/agent/agent_0.py

```diff
@@ -340,75 +349,74 @@ def _start_services(self):
         If a `./services` file exist, reserve a compute node and run that file
         there as bash script.
         '''
+        self._log.info('Starting the agent services')
+        service_descriptions = self._cfg.services
+        services = list()
+
+        for service_desc in service_descriptions:
+
+            td   = TaskDescription(service_desc)
+            cfg  = self._cfg
+            task = dict()
+            tid  = ru.generate_id('service.%(item_counter)04d',
+                                  ru.ID_CUSTOM, ns=self._cfg.sid)
+
+            task['origin']            = 'agent'
+            task['description']       = td.as_dict()
+            task['state']             = rps.AGENT_STAGING_INPUT_PENDING
+            task['status']            = 'NEW'
+            task['type']              = 'service_task'
+            task['uid']               = tid
+            task['pilot_sandbox']     = cfg.pilot_sandbox
+            task['task_sandbox']      = cfg.pilot_sandbox + task['uid'] + '/'
+            task['task_sandbox_path'] = cfg.pilot_sandbox + task['uid'] + '/'
+            task['session_sandbox']   = cfg.session_sandbox
+            task['resource_sandbox']  = cfg.resource_sandbox
+            task['pilot']             = cfg.pid
+            task['resources']         = {'cpu': td.ranks * td.cores_per_rank,
+                                         'gpu': td.ranks * td.gpus_per_rank}
+
+            self._service_task_ids.append(tid)
+            services.append(task)
+
+        self.number_of_services_to_launch = len(services)
```

Comment:
this is not needed and is not used later

Reply:
Fixed